index.js

```diff
@@ -1,13 +1,26 @@
 const puns = require("./puns.json");
 
+/**
+ * Get all puns.
+ * @returns {Array} Array of pun objects (pun and punchline)
+ */
 exports.all = () => puns;
 
+/**
+ * Get random pun.
+ * @returns {Object} Pun object (pun and punchline)
```

Comment:
That's not exactly JSDoc standard. I would expect Objects to be documented as:

/**
 * Get a random puns.
 * @param {Object} pun - The pun object.
 * @param {string} pun.pun - The pun.
 * @param {string} pun.punchline - The punchline.
 */